src/Microsoft.SqlTools.ServiceLayer/Localization/sr.resx

```diff
@@ -151,12 +151,16 @@
  Parameters: 0 - encrypt (string) </comment>
   </data>
   <data name="ConnectionServiceConnStringInvalidEnclaveAttestationProtocol" xml:space="preserve">
-    <value>Invalid value &apos;{0}&apos; for EnclaveAttestationProtocol. Valid values are &apos;AAS&apos; and &apos;HGS&apos;.</value>
+    <value>Invalid value &apos;{0}&apos; for EnclaveAttestationProtocol. Valid values are &apos;AAS&apos;, &apos;HGS&apos; and &apos;None&apos;.</value>

```

Comment:
Is "None" being displayed in the UI or will it be "NONE"?

Reply:
It will be displayed as 'None' to user, internally compared as 'NONE'